spec/models/chef_spec.rb

```diff
@@ -10,4 +10,17 @@
       it {should have_many :dishes}
   end
 
-end
\ No newline at end of file
+  describe "#ingredients_used" do
+    it "lists all the ingredients a chef uses" do
+      @chef1 = Chef.create(name: "Kam")
+      @dish1 = @chef1.dishes.create(name: "cereal au lait", description: "lucky charms in a whole milk reduction")
+      @milk = Ingredient.create(name: "whole milk", calories: 500)
+      @cereal = Ingredient.create(name: "lucky charms", calories: 400)
+      @extra_sugar = Ingredient.create(name: "extra sugar", calories: 20000)
+      @dish1.ingredients << @milk
+      @dish1.ingredients << @cereal
+
+      expect(@chef1.ingredients_used).to include(@cereal, @milk)
```

Comment:
If you pluck at the end of this query and test for the exact array of ingredients coming back, this will help test for duplicates in the return so you can catch that you need `distinct`!